jekyll/_cci2/api-intro.md

```diff
@@ -109,10 +108,10 @@ Endpoint       | Description
 
 API v2 is not currently supported for self-hosted installations of CircleCI Server.
 
-<!---
+## Data Insights (Preview)
+
 The CircleCI API v2 enables you to call a specific set of endpoints to retrieve detailed insights and data about your jobs and workflows. This information can be very useful in better understanding how your jobs and workflows are performing while also providing you with data points that you can use to optimize your workflows and builds. A detailed *API Reference Guide* (*add link here for the API Reference Guide when ready*) for these API endpoints has been provided in the documentation. Some examples of Insights API endpoints include:
```

Comment:
Can we add the link in here for the * content?

Reply:
Fixed